mbed_host_tests/host_tests_conn_proxy/conn_primitive_serial.py

```diff
@@ -84,6 +87,9 @@ def reset_dev_via_serial(self, delay=1):
 
     def read(self, count):
         """! Read data from serial port RX buffer """
+        # TIMEOUT: Since read is called in a loop, wait for self.timeout period before calling serial.read(). See
+        # comment on serial.Serial() call above about timeout.
+        sleep(self.timeout)
```

Comment:
But they will be impacted. Yes we print every 1 sec, but the host tests are constantly looking for drift between each tick. And the `sleep` call will prevent us from assigning a time stamp that is close to when we actually receive the serial data. This is the line I think that will be impacted: https://github.com/ARMmbed/htrun/pull/130/files#diff-9e6e1b40e765e6a2609375222e35117dR48

I agree that running timing tests on non-realtime  OS is not all that great, I was talking to @sg- about this. But at the moment will still do have these tests.